src/coreclr/jit/flowgraph.cpp

```diff
@@ -3626,6 +3617,12 @@ PhaseStatus Compiler::fgCreateThrowHelperBlocks()
         BasicBlock* const newBlk = fgNewBBinRegion(jumpKinds[add->acdKind], srcBlk, /* jumpDest */ nullptr,
                                                    /* runRarely */ true, /* insertAtEnd */ true);
 
+        if (newBlk->KindIs(BBJ_ALWAYS))
+        {
+            assert(add->acdKind == SCK_NONE);
+            newBlk->SetJumpDest(newBlk->Next());
```

Comment:
What if newBlk is the last block of the function? Then is `bbJumpDest == nullptr` an indication of "fall off the end?" (previously, we'd have a BBJ_NONE and generate an `int3` / breakpoint if that happened)

Reply:
Good point. When emitting jumps, I added in assert to see if we ever get a `BBJ_ALWAYS` with a null jump target, and it never hit during SuperPMI replays. Per your note on `SCK_NONE`, I think we can get rid of this if statement altogether.